src/containers/LogIn/LogIn.jsx

```diff
@@ -38,26 +39,36 @@ const theme = createTheme();
 
 const LogIn = () => {
   const dispatch = useDispatch();
+  const requestError = useSelector((state) => state.currentUser.error);
```

Comment:
You can call this with [] brackets . And at the end dont give the name (error) give it at the beggining 
const [error]  = useSelector((state) => state.currentUser);

This way you can call multiple variables from this reducer